src/calendar/calendar.module.ts

```
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { CalendarService } from './calendar.service';
+import { CalendarController } from './calendar.controller';
+import { ScheduleModule } from '@nestjs/schedule';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Promise } from 'src/promise/entity/promise.entity';
```

Biome (1.9.4)

[error] Do not shadow the global "Promise" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)